cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -167,7 +167,41 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
         assertStats(r)
       })){}
 
-     }
+      p("Privileges can also be output as Cypher commands with the optional `AS COMMAND[S]`.")
+      query("SHOW ROLE admin PRIVILEGES AS COMMANDS", assertPrivilegeShown(Seq(
+        Map("command" -> "GRANT ACCESS ON DATABASE * TO `admin`"),
+        Map("command" -> "GRANT ALL DBMS PRIVILEGES ON DBMS TO `admin`"),
+        Map("command" -> "GRANT CONSTRAINT MANAGEMENT ON DATABASE * TO `admin`"),
+        Map("command" -> "GRANT INDEX MANAGEMENT ON DATABASE * TO `admin`"),
+        Map("command" -> "GRANT MATCH {*} ON GRAPH * NODE * TO `admin`"),
+        Map("command" -> "GRANT MATCH {*} ON GRAPH * RELATIONSHIP * TO `admin`"),
+        Map("command" -> "GRANT NAME MANAGEMENT ON DATABASE * TO `admin`"),
+        Map("command" -> "GRANT START ON DATABASE * TO `admin`"),
+        Map("command" -> "GRANT STOP ON DATABASE * TO `admin`"),
+        Map("command" -> "GRANT TRANSACTION MANAGEMENT (*) ON DATABASE * TO `admin`"),
+        Map("command" -> "GRANT WRITE ON GRAPH * TO `admin`"),
+      ))) {
+        resultTable()
+      }
+
+      p("When showing _user_ privileges as commands, the roles in the Cypher commands will be replace with a parameter.")
+      query("SHOW USER jake PRIVILEGES AS COMMAND", assertPrivilegeShown(Seq(
+        Map("command" -> "GRANT ACCESS ON DATABASE `neo4j` TO $role"),
+        Map("command" -> "GRANT ACCESS ON DEFAULT DATABASE TO $role"),
+        Map("command" -> "GRANT EXECUTE PROCEDURE * ON DBMS TO $role"),
+      ))) {
+        resultTable()
+      }
+
+      p("It's also possible to have privileges output as revoke commands. " +
```

Comment:
Maybe add something like - 'Like other `SHOW` commands, the output can also be processed using YIELD / WHERE / RETURN'